mtools/mloginfo/sections/connection_section.py

```diff
@@ -1,5 +1,6 @@
 from base_section import BaseSection
 from collections import defaultdict
+from datetime import datetime, date, time
```

Comment:
This import doesn't appear to be needed. Not calling any date functions in the current PR?

Reply:
done